outplacement/models/outplacement.py

```diff
@@ -27,12 +27,12 @@ def _default_stage_id(self):
 
     @api.model
     def _read_group_employee_ids(self, employees, domain, order):
-        """ Always display all stages """
+        """ Always display all employees in users performing_operation """
         _logger.warn('group by employee domain %s order %s' % (domain, order))
```

Comment:
Consider changing log level to INFO, so its possible to suppress when we don't want to see it